baseplate/lib/prometheus_metrics.py

```diff
@@ -36,3 +41,19 @@ def getHTTPSuccessLabel(httpStatusCode: int) -> str:
     The HTTP success label is "true" if the status code is 2xx or 3xx, "false" otherwise.
     """
     return str(200 <= httpStatusCode < 400).lower()
+
+
+def is_metrics_enabled(raw_config: Dict[str, str]) -> bool:
+    cfg = config.parse_config(
+        raw_config,
+        {
+            "metrics": {
+                "enabled": config.Optional(config.Boolean),
+            }
+        },
+    )
+
+    if cfg.metrics.enabled is not None:
+        return cfg.metrics.enabled
+
+    return "metrics.tagging" in raw_config or "metrics.namespace" in raw_config
```

Comment:
If not specified by the client, metrics should be enabled.

Reply:
In this case we can skip `"metrics.tagging" in raw_config or "metrics.namespace" in raw_config` check and just `return True`, right?

Reply:
Yea, though I think we will merge this the way it is and update this later. We need to get this change out to clients.

Reply:
Actually we want to change this here to make the user upgrade story easier.
```suggestion
    return True
```